lib/taskana-core/src/main/java/pro/taskana/workbasket/api/WorkbasketService.java

```diff
@@ -232,13 +238,15 @@ List<WorkbasketSummary> getDistributionSources(String workbasketKey, String doma
    *     Workbasket}
    * @param targetWorkbasketId the {@linkplain Workbasket#getId() id} of the target {@linkplain
    *     Workbasket}
-   * @throws NotAuthorizedException if the current user doesn't have {@linkplain
+   * @throws MismatchedWorkbasketPermissionException if the current user doesn't have {@linkplain
    *     WorkbasketPermission#READ READ permission} for the source {@linkplain Workbasket}s
+   * @throws MismatchedRoleException if the current user doesn't have needed Role
```

Comment:
if the current user is not member of {@linkplain TaskanaRole#BUSINESS_ADMIN} or {@linkplain TaskanaRole#ADMIN}